### ISSUE TITLE
Compute dataset statistics to better inform decisions

### ISSUE THREAD

**Issue by MatthewJA** [chengsoonong/crowdastro#229]

**Comment by chengsoonong** [comment]
See comment on 31 March in issue #232

**Comment by MatthewJA** [comment]
How many extended/pointlike sources are in RGZ ∩ Franzen? ∩ Norris?

How many objects are in
- RGZ ∩ Norris ∩ extended?
- RGZ ∩ Franzen ∩ extended?
- RGZ ∩ Franzen ∩ extended ∩ labelled?

**Comment by MatthewJA** [comment]
- |RGZ ∩ Norris ∩ extended| = 81
- |RGZ ∩ Franzen ∩ extended| = 307
- |RGZ ∩ Franzen ∩ extended ∩ labelled| = 130

**Comment by MatthewJA** [comment]
Having what looks to be sensible data now, what statistics would be useful? I'd like to have a concrete list and it's been long enough that I can't quite remember what statistics we were talking about.

**Comment by chengsoonong** [comment]
Repeating comment from issue #232 


Some more data investigation for labels:

Report the RGZ labeller agreements. Are there any differences in the distribution in the four subsets (intersection with Franzen and/or Norris)?
Look at the 4 subsets which are intersections of two or more datasets, studying agreement and disagreement
It would be best to have the most agreed labels as the test set. I hope/expect that there would be about 550 of the 613 where a significant number of RGZ labellers agree. If this is the case, then we can use half of this as a test set (and the remaining half for training). Of course, we can take 10 random splits.

For the rest of the training set, if any data set has labelled it, then we can use them as data.

**Comment by MatthewJA** [comment]
Ah, forgot that; thanks!

**Comment by chengsoonong** [comment]
For the non-CNN features, plot a 30 bin histogram for each feature.
By comparing the histograms of the different parts of the Venn diagram, maybe we can spot some problems.

**Comment by MatthewJA** [comment]
Here are some numbers for the compact/resolved subsets of ATLAS-CDFS:

Compact and Norris-labelled: 385
Resolved and Norris-labelled: 223
Compact and RGZ-labelled: 1986
Resolved and RGZ-labelled: 380
Compact and labelled by both: 332
Resolved and labelled by both: 188

**Comment by MatthewJA** [comment]
Plotted the histograms and it looks like RGZ has similar (astro) features to the Norris subset.

**Comment by MatthewJA** [comment]
![plots](https://cloud.githubusercontent.com/assets/2681312/25417125/23e7f57a-2a85-11e7-97b0-3df09be5b44a.png)

Feature plots. Left to right: colour, colour, stellarity, stellarity, distance.

**Comment by MatthewJA** [comment]
(The other statistics matter less now that we are using the right ATLAS dataset.)
